### PR TITLE
Added Omnibox experiment explainer

### PR DESCRIPTION
**Pull request by erdraud**

### CODE REVIEW

File: explainers/Omnibox/explainer.md [hunk 1]
```diff
@@ -0,0 +1,217 @@
+
+# DevTools: Omnibox Commands
+
+Authors:
+ - [Erica Draud](https://github.com/herdraud), Program Manager
+
+## Status of this feature
+This feature is a proposed experiment. To track the project status, you can see the scenario here: [[Experiment] DUX Omnibox Commands](https://dev.azure.com/microsoft/Edge/_workitems/edit/29631932)
```
Comment (captainbrosset) [inline explainers/Omnibox/explainer.md:8]: This link only works for Microsoft employees. You might want to remove it.

File: explainers/Omnibox/explainer.md [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+## Introduction
+Earlier this year, the Chrome team released the [Commander]( https://bugs.chromium.org/p/chromium/issues/detail?id=1014639) as a text interface that provides a shortcut for common commands for browser users. The DevTools team used this as the framework for their Omnibar hackathon project. There’s some uncertainty around the usage and discoverability of the tool, so we’re starting with a flagged experiment.
+![Omnibox UI](omni.PNG)
+You’ll notice that many of the commands in this feature can be found in the [Command Palette]( https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/command-menu/) in DevTools, so why have these features in the address bar (from this point forward referred to as the omnibox)? We know that users would like a faster way to complete their tasks in DevTools, and the Omnibox Commands feature enables users to do so without having to first open the Developer Tools. Additionally, we’ve included features that are specific to the browser window: Edge internal urls and last viewed pages.
```
Comment (captainbrosset) [inline explainers/Omnibox/explainer.md:13]:
```suggestion
You’ll notice that many of the commands in this feature can be found in the [Command Menu]( https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/command-menu/) in DevTools, so why have these features in the address bar (from this point forward referred to as the omnibox)? We know that users would like a faster way to complete their tasks in DevTools, and the Omnibox Commands feature enables users to do so without having to first open the Developer Tools. Additionally, we’ve included features that are specific to the browser window: Edge internal urls and last viewed pages.
```

File: explainers/Omnibox/explainer.md [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+### Recently closed
+To get a list of the webpages you recently closed, type `#`.
+
+## Next steps
+* Design review
+* Flagged rollout
+* Add Documentation Search
+
+## Things to consider
+* Commands shouldn't conflict with Bing searches
```
Comment (captainbrosset) [inline explainers/Omnibox/explainer.md:40]:
```suggestion
* Commands shouldn't conflict with search engine searches
```

**Review by captainbrosset — APPROVED** [review]

File: explainers/DocumentationSearch/explainer.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+# DevTools: Documentation Search
+Authors:
+- [Erica Draud](https://github.com/erdraud), Program Manager
+- Sean Larkin
```
Comment (erdraud) [inline explainers/DocumentationSearch/explainer.md:4]: @TheLarkInn I used a lot of text from your document. I can keep your name and add your github account or remove your name. Let me know what you prefer.

File: explainers/Omnibox/explainer.md (hunk 1 above)
Comment (erdraud) [reply explainers/Omnibox/explainer.md]: Done!

File: explainers/DocumentationSearch/explainer.md (hunk 4 above)
Comment (TheLarkInn) [reply explainers/DocumentationSearch/explainer.md:4]: Whatever you prefer is fine. I don't mind either :+1:

**Comment by captainbrosset** [comment]
@erdraud do you think we can merge this? Do you need reviews from other people before merging?

**Comment by erdraud** [comment]
@captainbrosset I can't merge it myself, but I this is ready to go.

**Pull request merged**